### ISSUE TITLE
[timm][training] `levit_128`, `xcit_large_24_p8_224` fail accuracy

### ISSUE THREAD

**Issue by whitneywhtsang** [intel/intel-xpu-backend-for-triton#525]
`levit_128`, `xcit_large_24_p8_224` fail accuracy for all data types in training mode.

**Comment by whitneywhtsang** [comment]
Not regression, lowered priority.

**Comment by vlad-penkin** [comment]
The accuracy issue for levit_128 is still reproducible for all data types except for float32.

Env:
- pytorch is built from source, top of the main trunk
- triton xpu is built from source, top of the main trunk
- benchmark is built from source, top of the main trunk
- torchaudio is built from source, top of the main trunk
- torchvision  is built from source, top of the main trunk
- PyTorch Dependency Bundle 0.5.0
- Latest Rolling Driver

**Comment by vlad-penkin** [comment]
No updates in year. Closing ticket.
